cmd/gardenlet/app/app.go

```diff
@@ -689,6 +694,37 @@ func cleanupShootCoreManagedResource(ctx context.Context, seedClient client.Clie
 	return flow.Parallel(taskFns...)(ctx)
 }
 
+// TODO(rfranzke): Remove this code after gardener v1.92 has been released.
+func reconcileLabelsForPVCMigrations(ctx context.Context, log logr.Logger, seedClient client.Client) error {
+	labelMigrationPVCName := "disk-migration.monitoring.gardener.cloud/pvc-name"
+
+	persistentVolumeList := &corev1.PersistentVolumeList{}
+	if err := seedClient.List(ctx, persistentVolumeList, client.HasLabels{labelMigrationPVCName}); err != nil {
+		return fmt.Errorf("failed listing persistent volumes with label %s: %w", labelMigrationPVCName, err)
+	}
+
+	var taskFns []flow.TaskFn
+
+	for _, pv := range persistentVolumeList.Items {
+		persistentVolume := pv
```

Comment:
Nit: This should not longer be required with go 1.22 as far as I understand the release notes, but it also does not harm.